jscomp/ml/printlambda.ml

```diff
@@ -21,7 +21,7 @@ open Lambda
 
 let rec struct_const ppf = function
   | Const_base(Const_int n) -> fprintf ppf "%i" n
-  | Const_base(Const_char c) -> fprintf ppf "%C" c
+  | Const_base(Const_char i) -> fprintf ppf "%C" (Char.unsafe_chr i)
   | Const_base(Const_string (s, _)) -> fprintf ppf "%S" s
```

Comment:
this should be adapted, '%C' may not apply any more

Reply:
Thank you! I'm fixing this.